example_site/src/components/constants.js

```diff
@@ -56,3 +56,41 @@ export const partnerConfigurations = {
     [partnerPresets.default]: defaultConfiguration,
     [partnerPresets.spectrum]: spectrumConfiguration,
 };
+
+export const videos = [
```

Comment:
Do you think something `accountVideos` or `userVideos` might be more descriptive?